bescort.lic

```diff
@@ -289,12 +293,44 @@ class Bescort
     elsif args.hara_polo
       hara_polo(args.mode)
     elsif args.jolas
-      jolas(args.mode)       
+      jolas(args.mode)
+    elsif args.currach
+      currach(args.mode)
     end
   end
 
   private
 
+  def currach(mode)
+    if (mode.include?('aesry') && (Room.current.id == 5555 || Room.current.id == 5557 || Room.current.id == 5558) || (mode.include?('halasa') && Room.current.id == 15863))
```

Comment:
This conditional is a bit hard to read, in my opinion. What would you think about refactoring it?

A few ideas:

* two `if` blocks, one for aesry and one for halasa
* Using something like `[5555, 5557, 5558].include?(Room.current.id)`

Reply:
Great idea!